js/packages/mobile-wallet-adapter-protocol/src/types.ts

```diff
@@ -54,16 +60,33 @@ type Base64EncodedSignedTransaction = string;
 
 export type Base64EncodedTransaction = string;
 
+/**
+ * @deprecated Replaced by the 'chain' parameter, which adds multi-chain capability as per MWA 2.0 spec.
+ */
 export type Cluster = 'devnet' | 'testnet' | 'mainnet-beta';
 
+export type Chain = IdentifierString | Cluster;
+
 export type Finality = 'confirmed' | 'finalized' | 'processed';
 
 export type WalletAssociationConfig = Readonly<{
     baseUri?: string;
 }>;
 
 export interface AuthorizeAPI {
+    /**
+     * @deprecated Replaced by updated authorize() method, which adds MWA 2.0 spec support.
+     */
     authorize(params: { cluster: Cluster; identity: AppIdentity }): Promise<AuthorizationResult>;
+
+    authorize(params: { 
+        identity: AppIdentity;
+        chain?: Chain;
+        features?: IdentifierArray; 
+        addresses?: string[]; 
+        auth_token?: AuthToken; 
+        sign_in_payload?: SolanaSignInInput;
```

Comment:
What happens when a v1 dApp tries to authorize with `sign_in_payload` using a legacy wallet? IIUC, the backwards compatibility isn't handled and the wallet will try a normal authorize.

How can the dApp check if the wallet supports SIWS?

Reply:
answered offline:
> it would need to call getCapabilities, just as normal/native

Reply:
a dapp can either call `getCapabilities` before hand, or it can simply call `authorize` with a sign in payload and check for a sign in result in the response. 

In a separate PR I will address SIWS more completely.